shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -962,6 +984,34 @@ - (BOOL)isAlwaysUse24HourFormat {
   return [dateFormat rangeOfString:@"a"].location == NSNotFound;
 }
 
+- (NSString*)brightnessMode {
```

Comment:
Google's ObjC style requires declaration comments on all non-trivial methods, including private ones. Please comment this and the next method. A good thing to explain as part of that would be what these seemingly-arbitrary-in-this-context strings are (i.e., that they are values expected by the framework for the <insert key name> key in the settings channel).

http://google.github.io/styleguide/objcguide.html#declaration-comments

Reply:
done